adam-core/src/main/scala/org/bdgenomics/adam/models/SequenceDictionary.scala

```diff
@@ -184,7 +184,7 @@ class SequenceDictionary(val records: Vector[SequenceRecord]) extends Serializab
    * @return Returns a SAM formatted sequence dictionary.
    */
   def toSAMSequenceDictionary: SAMSequenceDictionary = {
-    new SAMSequenceDictionary(records.iterator.map(_.toSAMSequenceRecord).toList)
+    new SAMSequenceDictionary(records.map(_.toSAMSequenceRecord).asJava)
```

Comment:
This is a good catch, did you have profiling results leading you here, or were you able to eyeball it?

Reply:
Profiler, of course!